pkg/docker/docker.go

```diff
@@ -18,8 +18,8 @@ const (
 	ScriptsURLEnvironment = "STI_SCRIPTS_URL"
 	LocationEnvironment   = "STI_LOCATION"
 
-	ScriptsURLLabel  = "io.s2i.scripts-url"
```

Comment:
I'd leave this as a constant here, at least it's easier for reading, at most adding depecated comment, which should be also next to accompanying envs.